docs/UserGuide.adoc

```diff
@@ -112,44 +112,15 @@ Adds a contact named Joel HH with email address joel@yahoo.com.sg, phone number
 * `(ab)add n/Aisyle Nat e/aisyle@gmail.com p/87654321 o/NUS b/02-12` +
 Adds a contact named Aisyle Nat with email address aisyle@gmail.com, phone number 87654321 and birthday 12 February into the organization NUS.
 
-==== Deleting a person : `(ab)delete`
-
-Deletes the contact at index INDEX +
-Format: `(ab)delete INDEX [t/] [e/] [a/] [o/] [b/]`
-
-****
-* INDEX must be a positive integer. It refers to the index of the respective contact as shown in the displayed person list.
-* Fields inside square brackets are optional.
-* Specifying any of the optional fields will delete the existing values.
-* If none of the optional fields are specified, the entire contact will be deleted.
-****
-
-Examples:
-
-* `(ab)delete 2` +
-Deletes the 2nd person in the address book. +
-Format: `(ab)delete INDEX [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [o/ORGANIZATION] [t/TAG]...`
-* `(ab)delete 3 a/` +
-Deletes the third person's address.
-* `(ab)delete 20 e/ b/ i/
-Deletes the 20th person's email, birthday, and all remarks.
-
-****
-* Deletes the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided and cannot be left empty.
-* Name and phone of the contact cannot be deleted.
-* Existing values will be deleted if the field is specified.
-* All tags and remarks will be deleted if indicated.
-****
-
 ==== Editing a person : `(ab)edit`
 
 Edits an existing person in the address book. +
 Format: `(ab)edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [o/ORGANIZATION] [t/TAG_TO_BE_ADDED]... [-t/TAG_TO_BE_DELETED]...`
 
 ****
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided and parameter cannot be left empty.
+* At least one of the optional fields must be provided.
+* If a field is specified but the parameter is left empty, the entire field will be deleted.
```

Comment:
Prob might wanna include that this is excluding the t/ prefix which will throw error